CHANGELOG.md

```diff
@@ -13,6 +13,60 @@ Breaking changes:
 - [debug] renamed getter method `multiSesssion` to `multiSession` [#6698](https://github.com/eclipse-theia/theia/pull/6698)
 - [task] changed the data structure of `ProvidedTaskConfigurations.tasksMap` [#6718](https://github.com/eclipse-theia/theia/pull/6718)
 - [task] added `taskDefinitionRegistry` and `taskSourceResolver` to the constructor of `TaskRunQuickOpenItem` and `ConfigureBuildOrTestTaskQuickOpenItem` [#6718](https://github.com/eclipse-theia/theia/pull/6718)
+- Support VS Code icon and color theming. [#6475](https://github.com/eclipse-theia/theia/pull/6475)
```

Comment:
@eclipse-theia/ecd-theia-committers Here is a list of breaking changes. Is it clear what should be done to deal with them?

We plan it to land after 0.14 release that it was in for the entire next month and released as a part of 0.15.